appinfo/app.php

```diff
@@ -5,10 +5,12 @@
     'Files_Sharing' => 'files',
 ];
 
-\OCP\Util::addScript('files_rightclick', 'script');
-
 $eventDispatcher = \OC::$server->getEventDispatcher();
 
+$eventDispatcher->addListener(\OCP\AppFramework\Http\TemplateResponse::EVENT_LOAD_ADDITIONAL_SCRIPTS_LOGGEDIN, function() {
```

Comment:
@NastuzziSamy Can't we actually just load the app on the files app? Since we discussed that other right click handling should be done in the vue components library

Reply:
Or is the app also handling right click outside of the files app?

Reply:
I think it is fine. If we make a vue component, it will only work with vue projets

I will make the app work with specific css classes, if it is ok for you, I will merge this PR

Reply:
@juliushaertl